### PR TITLE
Add optional label to dontAutomergeX renovate presets

### PR DESCRIPTION
**Pull request by stefee**

### CODE REVIEW

File: dontAutomergeMajor.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "description": "Disables automerge for major updates.",
+  "major": {
+    "automerge": false,
```
Comment (stefee) [inline dontAutomergeMajor.json:4]: This is already the default behaviour in most cases, but there's no harm in being explicit, and it means we can have the same `labels` functionality as other cases

**Review by ernestohegi — APPROVED** [review]
I guess it's fine?

File: README.md [hunk 2]
```diff
@@ -9,12 +9,12 @@ renovate.json
...
   "extends": [
-    "github>CondeNast/renovate-config:unpublishSafe",
     "github>CondeNast/renovate-config:forceCondenastUpdates",
     "github>CondeNast/renovate-config:automergeMinor",
     "github>CondeNast/renovate-config:condenastStabilityDays",
-    "github>CondeNast/renovate-config:dontAutomergeNode",
-    "github>CondeNast/renovate-config:dontAutomergeTestFrameworks",
+    "github>CondeNast/renovate-config:dontAutomergeMajor(help wanted)",
+    "github>CondeNast/renovate-config:dontAutomergeNode(help wanted)",
+    "github>CondeNast/renovate-config:dontAutomergeTestFrameworks(help wanted)",
```
Comment (stefee) [inline README.md:17]: with Rocket we will want to use `:heart: help wanted` rather than just `help wanted`

**Comment by stefee** [comment]
I'm hoping this isn't a breaking change, but I actually have no idea.

**Pull request merged**